Improve messaging to OESL (via email) and to responders (via SMS)

evanpyke opened the pull request:
Responder emails autofilled as recipients in email dialog.  Also updated readme to match latest gcp and gapps versions.

evanpyke committed:
testing escaped brackets in readme

evanpyke committed:
gapps init workaround

evanpyke committed:
add gapps.config.json.template

evanpyke committed:
"Test as Add-on" moved menu dropdown

evanpyke committed:
Autofill responders as recipients in email dialog

andrew-nguyen commented:
This looks pretty good.  What version of gapps are we on now?

evanpyke commented:
Funny you ask.  I just checked my gapps and the github page.  I'm on 1.2.0, and as of 2 days ago they are deprecating because google developed clasp, which looks like their version of gapps+github.

andrew-nguyen commented:
Think it's worth just jumping over to clasp now that things are fresh in your mind?

evanpyke commented:
I'll take a look

evanpyke committed:
moving to clasp since gapps is deprecated

evanpyke commented:
Updated with clasp, much simpler setup now.

evanpyke committed:
add sms gateway email thru d4h's pager_email (to be populated)

evanpyke committed:
remove hardcoded email, changed to active user email

evanpyke committed:
populate sms-email gateways within "Populate Call Sheet" button

also added private.js to define DO_PHONE variable

evanpyke committed:
remove sms gateway from carpool sheet

Accidentally added gateway to carpool sheet, conflicts with "Have car" column, not needed anyway.

evanpyke committed:
Merge pull request #1 from evanpyke/sms

Populate SMS-Email Gateways into Call Sheet

andrew-nguyen commented:
@evanpyke Took a quick look through this and looks great.  Just to confirm what I'm looking at - you're using the same email dialog template for both the email to the OESL (and responders) as well as when messaging responders directly via email?  If so, we should probably rename that file to make it less confusing.

I'll see if I can test it in our staging environment over the next week or so.

andrew-nguyen commented:
@kbisset Do you have time to try and test the new clasp setup?

evanpyke committed:
Updated OESL-Email-Dialog* to Email-Dialog* for more versatility

evanpyke commented:
@andrew-nguyen Yes they were using the same template.  I updated the Email Dialog template and moved the short instruction/subtitle into code.js
